<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateStationModal</class>
 <widget class="QWidget" name="UpdateStationModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Станция</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="inputsLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="inputGroup_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Название</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameInput">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="inputGroup_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="cityLabel">
             <property name="text">
              <string>Город</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cityInput"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="inputGroup">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="countryLabel">
             <property name="text">
              <string>Страна</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="countryInput"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Сохранить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
